ember-toucan-styles/src/utils/colors.ts

```diff
@@ -122,6 +129,31 @@ export function getGraphColors(count = 11) {
   return swatch.filter((color) => color !== null);
 }
 
+// By default includes '1' neutral color at the end of the palette, similar to graph
+export function getChartColors(count = 11, { includeNeutrals = true } = {}) {
```

Comment:
Decided to make one change here, so the `getChartColors` returns the same shape of colors as `getGraphColors` by default (10 normal + 1 neutral). So both methods return the same number by default now.